test/unit/plugins/guests/linux/cap/reboot_test.rb

```diff
@@ -68,7 +68,7 @@
         it "reboots the vm" do
           allow(communicator).to receive(:execute)
 
-          expect(communicator).to receive(:execute).with(/systemctl reboot/, nil).and_return(0)
+          expect(communicator).to receive(:execute).with(/systemctl/, nil).and_return(0)
```

Comment:
I think it would be useful to retain the `reboot` bit just to ensure that's what the command is doing. You can update the pattern to allow other stuff and still match: `/systemctl.*?reboot/`